apps/suite-launch/task-definitions/mock-mountebank-2.json

```diff
@@ -0,0 +1,172 @@
+{
+    "family": "{{{test_name_unique}}}",
+    "executionRoleArn": "arn:aws:iam::210752607241:role/ecsTaskExecutionRole",
+    "taskRoleArn": "arn:aws:iam::210752607241:role/FirelensECS",
+    "networkMode": "bridge",
+    "containerDefinitions": [
+      {
+        "name": "fluent-bit",
+        "image": "{{{image-aws-for-fluent-bit}}}",
+        {{#if commands-aws-for-fluent-bit}}
+        "command": ["/bin/bash", "-c", "{{#each commands-aws-for-fluent-bit}}{{{this}}}; {{/each}}"],
+        {{/if}}
+        "cpu": 512,
+        "firelensConfiguration": {
+          "type": "fluentbit"
+        },
+        "environmentFiles": [
+          {
+              "value": "{{{s3_materials_arn}}}/fluent-bit.env",
+              "type": "s3"
+          }
+        ],
+        "environment": [
+          {
+            "name": "AWS_REGION",
+            "value": "us-west-2"
+          },
+          {
+            "name": "aws_fluent_bit_init_s3_1",
+            "value": "{{{s3_materials_arn}}}/fluent-bit.conf"
+          },
+          {
+            "name": "aws_fluent_bit_init_s3_2",
+            "value": "{{{s3_materials_arn}}}/fluent-bit-onepod.conf"
+          },
+          {
+            "name": "aws_fluent_bit_init_s3_3",
+            "value": "{{{s3_materials_arn}}}/fluent-parser.conf"
+          },
+          {
+            "name": "S3_BUCKET",
+            "value": "{{{s3-corefile-destination}}}"
+          },
+          {
+            "name": "S3_KEY_PREFIX",
+            "value": "{{{test_name}}}/r{{{test_revision}}}"
+          },
+          {
+            "name": "S3_FILE_NAME_PREFIX",
+            "value": "{{{test_name}}}-r{{{test_revision}}}"
+          },
+          {
+            "name": "S3_ALTERNATE_LOCATION_PREFIX",
+            "value": "{{{s3_materials_bucket}}}/{{{test_name}}}/error-reports/{{{test_name}}}-r{{{test_revision}}}"
+          }
+        ],
+        "memory": 512,
+        "essential": true,
+        "logConfiguration": {
+          "logDriver": "awslogs",
+          "options": {
+            "awslogs-group": "app-{{{test_name_unique}}}",
+            "awslogs-region": "us-west-2",
+            "awslogs-create-group": "true",
+            "awslogs-stream-prefix": "fluent-bit"
+          }
+        },
+        "mountPoints": [
+          {
+            "sourceVolume": "my-shared-volume",
+            "containerPath": "/tmp/data_logs",
+            "readOnly": false
+          }
+        ],
+        "links": ["mock:mock"]
```

Comment:
why are we using links? If we run in awsvpc mode all containers can talk over localhost. I guess I see you have the mode set to bridge- why? Aren't we using fargate?

Reply:
I guess this is the version of it for EC2? If so, why is it named with `-2` when the next one has `-fargate`?

Reply:
Uh yes... Though it would probably be better to rename it to -ec2...